apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -169,7 +169,7 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit, OnDestroy {
                 }
             }
             this.salesService.getInvoiceList(requestObject, this.invoiceFormDetails.voucherDetails.voucherDate).pipe(takeUntil(this.destroyed$)).subscribe((response) => {
-                if (response && response.body) {
+                if (response?.body?.results?.length > 0) {
```

Comment:
This will prevent the old results from getting reset if the fresh results have `length` of `0`